index.d.ts

```diff
@@ -1,6 +1,11 @@
 import { FastifyPluginCallback } from 'fastify'
+import { EventEmitter } from 'events'
 
-export type fastifyGracefulShutdownOpt = { timeout?: number }
+export type fastifyGracefulShutdownOpt = {
+  timeout?: number
+  resetHandlersOnInit?: boolean
+  handlerEventListener?: EventEmitter & { exit(code?: number): never; }
```

Comment:
Could you use void here? Never is usually used for something else https://www.tutorialsteacher.com/typescript/typescript-never

Reply:
this was copied from node.js exit definition, I think. technically it will never return

Reply:
You're right. I overlooked it.